Fix issue #21

SantiNoccetti opened the pull request:
Close #21

SantiNoccetti committed:
Se agrega el atributo 'dni' en la clase Person y se modifican los metodos de los archivos para su funcionamiento

SantiNoccetti committed:
Agrega metodos para exportar tablas en formato CSV (#24)

* Agrega metodos para exportar tablas en formato CSV

* Mueve exportTableToCSV() a privado

* agrega eleccion el archivo de salida

---------

Co-authored-by: corte <cortesiniluciano@gmail.com>

Corte0 commented on Person.cpp:
```diff
@@ -1,33 +1,38 @@
-std::string Person::getAddress() const {
+unsigned int Person::getDni() const{
+    return dni;
+}
```
dejar espacio ahi

Corte0 reviewed (commented):
Tiene buena pinta, pone el espacio que te marque, proba bien todo, y le mandamos

Corte0 committed:
Create compilationCheck.yml (#27)

Corte0 committed:
Update compilationCheck.yml (#28)

Ignagx committed:
Fix issue #6 (#26)

* Agrega constructor de copia

* Permite que cada cliente tenga mas de un vehiculo

Se cambia el objeto vehicle a vehicleVector,
siendo un vector, el cliente puede tener mas de un
vehiculo. Ademas, agrega los metodos addVehicle
y removeVehicle, permitiendo la permutacion del
vector.

* Borra el constructor default

Se habia confundido el constructor de copia con el default. Al final esa linea no hacia nada.

* Crea metodo para eliminar vehiculos por license

Se cambia el nombre de removeVehicle a removeVehicleByLicense

* Agrega valor int devuelto en removeVehicleByPos

Se agrega que el metodo devuelva un valor de tipo entero para deteccion de errores

* Evita que se creen vehiculos vacios al instanciar

* Permite el manejo de varios vehiculos por cliente

Crea el metodo addMultipleVehicles() para agregar varios vehiculos a un cliente
Ademas, modifica los metodos getClientByName() y getClientById para que pueda
devolver apropiadamente la el objeto con todos los vehiculos

* Maneja error de vehiculo inexistente al borrar

* Borra archivos comiteados por accidente

* Cambia valor devuelto cuando no hay error

* Borra Sapei.db, agregado por accidente

* Revierte cambios de manejo de errores

Se hacen estos cambios en fix-issue-#30

* Elimina variabel no utilizada

* Elimina comentario de dudosa procedencia

---------

Co-authored-by: corte <cortesiniluciano@gmail.com>

Corte0 committed:
Update README.md (#29)

SantiNoccetti committed:
fix-issue-#21

SantiNoccetti committed:
fix-issue-#21
